internal/commands/download_sbom.go

```diff
@@ -0,0 +1,41 @@
+package commands
+
+import (
+	"github.com/spf13/cobra"
+
+	cpkg "github.com/buildpacks/pack/pkg/client"
+	"github.com/buildpacks/pack/pkg/logging"
+)
+
+type DownloadSBOMFlags struct {
+	Local          bool
+	Remote         bool
+	DestinationDir string
+}
+
+func DownloadSBOM(
+	logger logging.Logger,
+	client PackClient,
+) *cobra.Command {
+	var flags DownloadSBOMFlags
+	cmd := &cobra.Command{
+		Use:     "download-sbom <image-name>",
+		Args:    cobra.ExactArgs(1),
+		Short:   "Download SBoM from specified image",
+		Example: "pack download-sbom buildpacksio/pack",
+		RunE: logError(logger, func(cmd *cobra.Command, args []string) error {
+			img := args[0]
+			options := cpkg.DownloadSBOMOptions{
+				Daemon:         !flags.Remote,
+				DestinationDir: flags.DestinationDir,
+			}
+
+			return client.DownloadSBOM(img, options)
+		}),
+	}
+	AddHelpFlag(cmd, "download-sbom")
+	cmd.Flags().BoolVar(&flags.Local, "local", false, "Pull SBoM from local daemon (Default)")
```

Comment:
It looks like `local` isn't even used. Do you think we need it?

Reply:
From a code perspective, it's definitely not needed. I assumed this would make sense from a user perspective though: being explicit about `--local`, in the same way that you are explicit about `--remote`, without having to guess what the behavior would be.

I'm not married to it though. Happy to switch it if you have better ideas.

Reply:
I personally don't think it adds so much. It's an extra affordance that doesn't really do much, but makes users think it does. I'd suggest taking it out, and saying in the help that the base is that it just inspects a local image